apps/pie-microsite/src/_11ty/shortcodes/tokensTable/index.js

```diff
@@ -119,19 +131,23 @@ const buildTokenListElements = ({
     tokenType,
     tokenScssName,
     tokenDisplayName,
-    tokenMetadata
+    tokenMetadata = {}
 }) => {
     const tokenPill = buildTokenPill(tokenScssName);
     const tokenExampleElement = buildTokenExampleElement(token, tokenType);
-
+    
     // TODO - description is just an example of how we might use the metadata
     // We would likely wanted to move them into a colour specific handler similar to how we build
     // the colour token example. Please consider them placeholder for now.
     const tokenDescription = tokenMetadata.description
         ? `<span class="c-tokensTable-tokenDescription">${tokenMetadata.description}</span>`
         : '';
+    
+    const cssVariable = tokenType === 'color'
+        ? '--token-row-height: 128px'
+        : '--token-row-height: 176px';
 
-    return `<li class="c-tokensTable-row c-tokensTable-item">
+    return `<li class="c-tokensTable-row c-tokensTable-item" style="${cssVariable}">
```

Comment:
ah I see, I was wondering if in the SCSS we could default to 176px and if its colour then use 128. It would save us having to inject this to every node on the page